docs/pages/component/props.mdx

```diff
@@ -437,10 +437,23 @@ Determine whether the media should continue playing when notifications or the Co
 
 An image to display while the video is loading
 
-Value: string with a URL for the poster, e.g. "https://baconmockup.com/300/200/"
+Value: Props for the `Image` component. The poster is visible when the source attribute is provided.
+> [!WARNING]
```

Comment:
Can you please move the Warning under the title to keep doc uniform in doc

Reply:
Sure ;))